### ISSUE TITLE
Crash: Cursor.getString(1) must not be null 

### ISSUE THREAD

**Issue by ArtexJay** [yanus171/Handy-News-Reader#854]
**Describe the bug**
Crashes during import now, won't stop crashing when the app is opened

UNCAUGHT EXCEPTION
-------------------
Stacktrace:
java.lang.NullPointerException: cursor.getString(1) must not be null
	at ru.yanus171.feedexfork.utils.EntryUrlVoc.init_(EntryUrlVoc.kt:63)
	at ru.yanus171.feedexfork.utils.EntryUrlVoc.initInThread$lambda$4(EntryUrlVoc.kt:48)
	at ru.yanus171.feedexfork.utils.EntryUrlVoc.$r8$lambda$YSI1G6vEPcQBD-tgBdhXNWx-I3U(Unknown Source:0)
	at ru.yanus171.feedexfork.utils.EntryUrlVoc$$ExternalSyntheticLambda0.run(Unknown Source:0)
	at java.lang.Thread.run(Thread.java:923)

-------------------
Version : 0.18.12
Version Code: 293
Package : ru.yanus171.feedexfork
FilePath : /data/user/0/ru.yanus171.feedexfork/files
Phone Model: Mi 9T Pro
Android Version : 11
Board : raphael
Brand : Xiaomi
Device : raphael
Model : Mi 9T Pro
Product : raphael
User : builder
Total Internal memory : 117402300416
Available Internal memory : 11538927616
-------------------

**To Reproduce**
1. Import from backup file

**Expected behavior**
Correctly import the feeds, articles, etc from the backup file

**Comment by yanus171** [comment]
Try the last beta v0.18.13


TheEmperorArt ***@***.***> 25 февраля 2023 г. 23:50:10 написал:

**Comment by ArtexJay** [comment]
UNCAUGHT EXCEPTION
-------------------
Stacktrace:
java.lang.NullPointerException: cursor.getString(1) must not be null
	at ru.yanus171.feedexfork.utils.EntryUrlVoc.init_(EntryUrlVoc.kt:63)
	at ru.yanus171.feedexfork.utils.EntryUrlVoc.initInThread$lambda$4(EntryUrlVoc.kt:48)
	at ru.yanus171.feedexfork.utils.EntryUrlVoc.$r8$lambda$YSI1G6vEPcQBD-tgBdhXNWx-I3U(Unknown Source:0)
	at ru.yanus171.feedexfork.utils.EntryUrlVoc$$ExternalSyntheticLambda0.run(Unknown Source:0)
	at java.lang.Thread.run(Thread.java:923)

-------------------
Version : 0.18.13
Version Code: 294
Package : ru.yanus171.feedexfork
FilePath : /data/user/0/ru.yanus171.feedexfork/files
Phone Model: Mi 9T Pro
Android Version : 11
Board : raphael
Brand : Xiaomi
Device : raphael
Model : Mi 9T Pro
Product : raphael
User : builder
Total Internal memory : 117402300416
Available Internal memory : 11520671744
-------------------

Exact same issue.

**Comment by yanus171** [comment]
Try https://github.com/yanus171/Handy-News-Reader/issues/683#issuecomment-1445433229

**Comment by ArtexJay** [comment]
I am now getting a new error when importing and the app continues to crash immediately after launching it again

UNCAUGHT EXCEPTION
-------------------
Stacktrace:
java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter link
	at ru.yanus171.feedexfork.utils.FileUtils.getLinkHash(Unknown Source:2)
	at ru.yanus171.feedexfork.service.FetcherService.deleteGhost(FetcherService.java:528)
	at ru.yanus171.feedexfork.service.FetcherService.access$100(FetcherService.java:194)
	at ru.yanus171.feedexfork.service.FetcherService$1.run(FetcherService.java:368)
	at ru.yanus171.feedexfork.service.FetcherService.LongOper(FetcherService.java:615)
	at ru.yanus171.feedexfork.service.FetcherService.LongOper(FetcherService.java:604)
	at ru.yanus171.feedexfork.service.FetcherService.onHandleIntent(FetcherService.java:360)
	at android.app.IntentService$ServiceHandler.handleMessage(IntentService.java:78)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.os.HandlerThread.run(HandlerThread.java:67)

-------------------
Version : 0.18.14
Version Code: 295
Package : ru.yanus171.feedexfork
FilePath : /data/user/0/ru.yanus171.feedexfork/files
Phone Model: Mi 9T Pro
Android Version : 11
Board : raphael
Brand : Xiaomi
Device : raphael
Model : Mi 9T Pro
Product : raphael
User : builder
Total Internal memory : 117402300416
Available Internal memory : 11520671744
-------------------

**Comment by yanus171** [comment]
Try https://github.com/yanus171/Handy-News-Reader/issues/683#issuecomment-1484184430

**Comment by ItsIgnacioPortal** [comment]
This issue still happens in the latest beta :(

```
06-10 06:14:23.695  4537  4558 E HandyNewsLog: UNCAUGHT EXCEPTION
06-10 06:14:23.695  4537  4558 E HandyNewsLog: -------------------
06-10 06:14:23.695  4537  4558 E HandyNewsLog: Stacktrace:
06-10 06:14:23.695  4537  4558 E HandyNewsLog: java.lang.NullPointerException: cursor.getString(1) must not be null
06-10 06:14:23.695  4537  4558 E HandyNewsLog:  at ru.yanus171.feedexfork.utils.Label.<init>(LabelVoc.kt:46)
06-10 06:14:23.695  4537  4558 E HandyNewsLog:  at ru.yanus171.feedexfork.utils.LabelVoc.init_(LabelVoc.kt:72)
06-10 06:14:23.695  4537  4558 E HandyNewsLog:  at ru.yanus171.feedexfork.utils.LabelVoc.initInThread$lambda$15(LabelVoc.kt:197)
06-10 06:14:23.695  4537  4558 E HandyNewsLog:  at ru.yanus171.feedexfork.utils.LabelVoc.$r8$lambda$WvNaf74W2KHgPM7pjyZ0i05C-Ec(Unknown Source:0)
06-10 06:14:23.695  4537  4558 E HandyNewsLog:  at ru.yanus171.feedexfork.utils.LabelVoc$$ExternalSyntheticLambda1.run(Unknown Source:0)
06-10 06:14:23.695  4537  4558 E HandyNewsLog:  at java.lang.Thread.run(Thread.java:764)
06-10 06:14:23.695  4537  4558 E HandyNewsLog:
06-10 06:14:23.695  4537  4558 E HandyNewsLog: -------------------
06-10 06:14:23.695  4537  4558 E HandyNewsLog: Version : 0.18.21
06-10 06:14:23.695  4537  4558 E HandyNewsLog: Version Code: 303
06-10 06:14:23.695  4537  4558 E HandyNewsLog: Package : ru.yanus171.feedexfork
06-10 06:14:23.695  4537  4558 E HandyNewsLog: FilePath : /data/user/0/ru.yanus171.feedexfork/files
06-10 06:14:23.695  4537  4558 E HandyNewsLog: Phone Model: ONEPLUS A5000
06-10 06:14:23.695  4537  4558 E HandyNewsLog: Android Version : 9
06-10 06:14:23.695  4537  4558 E HandyNewsLog: Board : msm8998
06-10 06:14:23.695  4537  4558 E HandyNewsLog: Brand : OnePlus
06-10 06:14:23.695  4537  4558 E HandyNewsLog: Device : OnePlus5
06-10 06:14:23.695  4537  4558 E HandyNewsLog: Model : ONEPLUS A5000
06-10 06:14:23.695  4537  4558 E HandyNewsLog: Product : OnePlus5
06-10 06:14:23.695  4537  4558 E HandyNewsLog: User : build
06-10 06:14:23.695  4537  4558 E HandyNewsLog: Total Internal memory : 67506212864
06-10 06:14:23.695  4537  4558 E HandyNewsLog: Available Internal memory : 60610818048
06-10 06:14:23.695  4537  4558 E HandyNewsLog: -------------------
06-10 06:14:23.695  4537  4558 W System.err: java.lang.NullPointerException: cursor.getString(1) must not be null
06-10 06:14:23.695  4537  4558 W System.err:    at ru.yanus171.feedexfork.utils.Label.<init>(LabelVoc.kt:46)
06-10 06:14:23.695  4537  4558 W System.err:    at ru.yanus171.feedexfork.utils.LabelVoc.init_(LabelVoc.kt:72)
06-10 06:14:23.695  4537  4558 W System.err:    at ru.yanus171.feedexfork.utils.LabelVoc.initInThread$lambda$15(LabelVoc.kt:197)
06-10 06:14:23.695  4537  4558 W System.err:    at ru.yanus171.feedexfork.utils.LabelVoc.$r8$lambda$WvNaf74W2KHgPM7pjyZ0i05C-Ec(Unknown Source:0)
06-10 06:14:23.695  4537  4558 W System.err:    at ru.yanus171.feedexfork.utils.LabelVoc$$ExternalSyntheticLambda1.run(Unknown Source:0)
06-10 06:14:23.695  4537  4558 W System.err:    at java.lang.Thread.run(Thread.java:764)
06-10 06:14:23.698  1904  2378 D ClipboardService: BstHostCallManagerService setClipboardText method returned: 0
06-10 06:14:23.698  1904  1936 I HCALL   : hcallSetClipboardTextRpc(UNCAUGHT EXCEPTION
06-10 06:14:23.698  1904  1936 I HCALL   : -------------------
06-10 06:14:23.698  1904  1936 I HCALL   : Stacktrace:
06-10 06:14:23.698  1904  1936 I HCALL   : java.lang.NullPointerException: cursor.getString(1) must not be null
06-10 06:14:23.698  1904  1936 I HCALL   :      at ru.yanus171.feedexfork.utils.Label.<init>(LabelVoc.kt:46)
06-10 06:14:23.698  1904  1936 I HCALL   :      at ru.yanus171.feedexfork.utils.LabelVoc.init_(LabelVoc.kt:72)
06-10 06:14:23.698  1904  1936 I HCALL   :      at ru.yanus171.feedexfork.utils.LabelVoc.initInThread$lambda$15(LabelVoc.kt:197)
06-10 06:14:23.698  1904  1936 I HCALL   :      at ru.yanus171.feedexfork.utils.LabelVoc.$r8$lambda$WvNaf74W2KHgPM7pjyZ0i05C-Ec(Unknown Source:0)
06-10 06:14:23.698  1904  1936 I HCALL   :      at ru.yanus171.feedexfork.utils.LabelVoc$$ExternalSyntheticLambda1.run(Unknown Source:0)
06-10 06:14:23.698  1904  1936 I HCALL   :      at java.lang.Thread.run(Thread.java:764)
06-10 06:14:23.698  1904  1936 I HCALL   :
06-10 06:14:23.698  1904  1936 I HCALL   : -------------------
06-10 06:14:23.698  1904  1936 I HCALL   : Version : 0.18.21
06-10 06:14:23.698  1904  1936 I HCALL   : Version Code: 303
06-10 06:14:23.698  1904  1936 I HCALL   : Package : ru.yanus171.feedexfork
06-10 06:14:23.698  1904  1936 I HCALL   : FilePath : /data/user/0/ru.yanus171.feedexfork/files
06-10 06:14:23.698  1904  1936 I HCALL   : Phone Model: ONEPLUS A5000
06-10 06:14:23.698  1904  1936 I HCALL   : Android Version : 9
06-10 06:14:23.698  1904  1936 I HCALL   : Board : msm8998
06-10 06:14:23.698  1904  1936 I HCALL   : Brand : OnePlus
06-10 06:14:23.698  1904  1936 I HCALL   : Device : OnePlus5
06-10 06:14:23.698  1904  1936 I HCALL   : Model : ONEPLUS A5000
06-10 06:14:23.698  1904  1936 I HCALL   : Product : OnePlus5
06-10 06:14:23.698  1904  1936 I HCALL   : User : build
06-10 06:14:23.698  1904  1936 I HCALL   : Total Internal memory : 67506212864
06-10 06:14:23.698  1904  1936 I HCALL   : Available Internal memory : 60610818048
06-10 06:14:23.698  1904  1936 I HCALL   : -------------------
06-10 06:14:23.698  1904  1936 I HCALL   : )
```
